### PR TITLE
adding basic exec commands for node as alias

### PR DESCRIPTION
**Pull request by ChrisB9**

### CODE REVIEW

File: .additional_bashrc.sh [hunk 1]
```diff
@@ -10,6 +10,8 @@ NODE_CONTAINER=$(sudo docker ps -f "name=${DOCKER_COMPOSE_PROJECT}_node_1" --for
 alias node='sudo docker exec -u1000 -w $(pwd) -it ${NODE_CONTAINER} node'
 alias npm='sudo docker exec -u1000 -w $(pwd) -it ${NODE_CONTAINER} npm'
 alias yarn='sudo docker exec -u1000 -w $(pwd) -it ${NODE_CONTAINER} yarn'
+alias node_exec='sudo docker exec -u1000 -w $(pwd) -it ${NODE_CONTAINER}'
```
Comment (Kanti) [inline .additional_bashrc.sh]: needs Documentation

Comment (Cyb10101) [reply .additional_bashrc.sh]: Please see: https://github.com/pluswerk/php-dev/pull/25

Comment (ChrisB9) [reply .additional_bashrc.sh]: I'll add documentation

- regarding #25 - there we encountered some minor problems, the group that node is using should be www-data, else the nginx/apache might have difficulties handling builded files that is being processed afterwards by php-fpm

Comment (Kanti) [reply .additional_bashrc.sh]: > I'll add documentation
> 
> * regarding #25 - there we encountered some minor problems, the group that node is using should be www-data, else the nginx/apache might have difficulties handling builded files that is being processed afterwards by php-fpm

see #34

**Pull request merged**